### DISCUSSION TITLE
Symmetrization switched off for Path - does Berry curvature generated along a path does not obey the symmetries in system?

### DISCUSSION THREAD

**Discussion by Bhuvaneswari-R432** [wannier-berri/wannier-berri#235] [Q&A]
When calculating Berry curvature along a path for a system and trying to get a fat-band of Berry curvature, wannierberri prints the following lines during its execution

All calculators are compatible
Symmetrization switched off for Path
The set of k points is a Path() with 215 points and labels {0: 'G', 70: 'H', 130: 'P', 165: 'N', 214: 'G'}
WARNING : symmetry is not used for a tabulation along path
generating K_list
Done
Done, sum of weights:215.0
symgroup : None

Does it imply that symmetry constraints of a system are not included in calculating Berry curvature along a path? Kindly let me know.

Regards,
Bhuvaneswari R

**Comment by stepan-tsirkin** [comment]
Hi. The symmetries are invoked in WannierBerri in two ways. First it was implemented such that the result of the calculation is symmetrized (no matter if the system may actually break some symmetries). This works when  `run( ..., 'symmetrize = True',...) ` and calculation  is done on a grid, but not on a Path.

If you used the symmetrization of the system ([`System.symmetrize()`](https://github.com/wannier-berri/wannier-berri/blob/5b509513b3e8493d23777dbe219dbec245ad510b/wannierberri/system/system.py#L171-L193 ) )  than it still takes effect on the calculation on a Path

**Reply by Bhuvaneswari-R432** [reply]
Thank you so much for the clarification, sir.
